Rakesh9100/CalcDiverse
# Added Shoe Size Calculator

# Fixes Issue🛠️
Closes #766

# Description👨‍💻 
The Shoe Size Calculator helps you find your perfect fit. Just enter your foot length (size) and you will get your UK, US, EU size .
accurately.

# Type of change📄



# How this has been tested✅

**Initial Setup**
 Load the calculator page (index.html) in a web browser.

**Testing Steps:**
Step 1: Enter a number in the "Enter shoe size" input field.
Step 2: Select options in both "Original Size" and "Original Unit" dropdown menus.
Step 3: Click on the "Convert" button to perform a conversion and display the result.
Step 4: Verify that the conversion result is displayed correctly in the #result area.

# Checklist✅ 

- [x] I have added demonstration in the form of GIF/video file
- [x] I am an Open Source Contributor

# Screenshots/GIF📷
![Screenshot 2024-05-27 133914](https://github.com/Rakesh9100/CalcDiverse/assets/99855497/301c9a4e-d5dc-46ae-9dfc-500323c7b378)
